spec.mediawiki

```diff
@@ -487,6 +585,127 @@ reconnect or to act at the appropriate time is considered a violation of
 [[#Rules_of_Community_Conduct|community conduct rules]], so can result in 
 penalties.
 
+===Order Book Subscriptions===
+
+An order book can be viewed and tracked by subscribing to a market. 
+
+'''JSON-RPC method:''' <code>orderbook</code>, '''originator: ''' client
+
+{|
+! field       !! type   !! description
+|-
+| basemarket  || string || currency code for the market's base asset
+|-
+| quotemarket || string || currency code for the market's quote asset
+|}
+
+The response will contain the complete market order book. 
+The order book and all updates include a '''sequence ID''', which increments by 
++1 for every valid order the DEX receives. The client is responsible for 
```

Comment:
The sequence ID would also have to increment when the server forcibly cancels orders without a cancel order from the client (e.g. disconnected clients)